src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -35,6 +35,12 @@ import ContentListFilters from './ContentListFilters';
 import Hide from '../Hide/Hide';
 import EmptyTableState from './components/EmptyTableState';
 import { useQueryClient } from 'react-query';
+import StatusIcon from './components/StatusIcon';
+
+export const red = global_danger_color_100.value;
+export const green = global_success_color_100.value;
+export const yellow = '#F0AB00';
```

Comment:
Can you not find these colors within [patternfly](https://www.patternfly.org/v4/guidelines/colors) that are matching this?

Reply:
I got these values from patternfly, but I did not find a global variable for them like red and green